lib/Jsrt/Jsrt.cpp

```diff
@@ -1415,7 +1415,8 @@ CHAKRA_API JsInstanceOf(_In_ JsValueRef object, _In_ JsValueRef constructor, _Ou
         VALIDATE_INCOMING_REFERENCE(constructor, scriptContext);
         PARAM_NOT_NULL(result);
 
-        *result = Js::RecyclableObject::FromVar(constructor)->HasInstance(object, scriptContext) ? true : false;
+        Js::Var booleanValue = Js::JavascriptOperators::OP_IsInst(object, constructor, scriptContext, nullptr);
+        *result = Js::JavascriptBoolean::FromVar(booleanValue)->GetValue() ? true : false;
```

Comment:
A little clunky but alright. You could write this as `!!Js::JavascriptBoolean::FromVar(booleanValue)->GetValue()`, though I don't really like this syntax.

Reply:
I'm open to change it, I just lifted this from JsBooleanToBool later in this file.